### PR TITLE
Remove sdk content from readme

### PR DESCRIPTION
**Pull request by mbenita-Cyberark**
added link to SDK repo

### Desired Outcome

SDK is now leaving under a different repo.
we should remove it from our docs and add reference to it in the README.md file

### Implemented Changes

- Removed sdk content from readme
- Added link to SDK repo
- Updated changelog

#### Changelog

- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [x] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [ ] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Removed
+- `SDK` with all it contents was removed and now maintained under https://github.com/cyberark/conjur-api-python
```
Comment (shulifink) [inline CHANGELOG.md:10]:
```suggestion
- `SDK` with all it contents was removed and is now maintained in the https://github.com/cyberark/conjur-api-python repository
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # conjur-api-python3
 
-This repository includes both the self-contained Conjur CLI (`conjur`) and the Python3-based SDK for
+This repository includes self-contained Conjur CLI (`conjur`) for
 accessing the Conjur API to manage Conjur resources.
 
+Note: Conjur python SDK was removed from this repo and is now maintained under
```
Comment (shulifink) [inline README.md:6]:
```suggestion
Note: The Conjur Python SDK was removed from this repo and is now maintained in the 
```

**Review by shulifink — CHANGES_REQUESTED** [review]
Good idea!

**Review by InbalZilberman — COMMENTED** [review]
LGTM

**Review by eladkug — DISMISSED** [review]
LGTM

**Comment by orenbm** [comment]
@mbenita-Cyberark please squash these commits before merging so that we don't have a `merge from main` commit in the git history

**Review by orenbm — APPROVED** [review]

**Pull request merged**